examples/data-channels-create/main.go

```diff
@@ -33,7 +33,7 @@ func main() {
 
 	// Set the handler for ICE connection state
 	// This will notify you when the peer has connected/disconnected
-	peerConnection.OnICEConnectionStateChange = func(connectionState ice.ConnectionState) {
+	peerConnection.OnIceConnectionStateChange = func(connectionState ice.ConnectionState) {
```

Comment:
This stuff we talked about in terms of the "ICE" vs "Ice" to be consistent. I guess again I am just being consistent. Generally speaking though I am really not as scared of breaking things for users much right now because we are not at 80% coverage and high spec compliance. I think when there are loads of dependent project is when we start worrying but as it stands right now, only recently people are started to adopt us according to https://godoc.org/github.com/pions/webrtc?importers. I really do believe that as long as we are improving in the direction of better readability people will make the right adjustments.